src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -448,24 +466,25 @@ EOF
           set day1 = `echo $time_steps[1] | cut -c1-8`
           @ NAVAIL = ($dayl - $day1) + 1
    
-          # not enough days? move on to the next collection
+          # not enough days for monthly mean? move on to the next collection
           if($NAVAIL != $NDAYS) continue
-      
-          # create the monly average
+   
+          # create monthly-mean nc4 file
+          #   input time stamps:  *.YYYYMMDD_HHMM.*  daily avg written directly by HISTORY.rc
+          #                       *.YYYYMMDD.*       daily file from concatenation of sub-daily files
 
-          ncra -h $EXPID.$ThisCol.${YYYY}${MM}??.nc4 ${EXPID}.${ThisCol}.monthly.$YYYY$MM.nc4
+          ncra -h $EXPID.$ThisCol.${YYYY}${MM}*.nc4 ${EXPID}.${ThisCol}.monthly.$YYYY$MM.nc4
           
-          # don't want a daily? delete the daily and sub-dailies and continue
-          #
-          if($NODAILIES == 2) then
+          if($POSTPROC_HIST == 2) then
+             # delete daily files
```

Comment:
check two things here:
1) does the desired monthly-mean file exist
2) is it non-zero size
if so, proceed with deletion of daily files, otherwise leave daily files in place and move on